tests/custom_kat.rs

```diff
@@ -0,0 +1,33 @@
+// Test that a custom handler works on wasm32-unknown-unknown.
+#![cfg(all(
+    target_arch = "wasm32",
+    target_os = "unknown",
+    feature = "custom",
+    not(feature = "js")
+))]
+
+use wasm_bindgen_test::wasm_bindgen_test as test;
+#[cfg(feature = "test-in-browser")]
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+mod custom_common;
+
+use custom_common::len7_err;
+use getrandom::getrandom;
+
+// This known-answer test cannot be in the same test suite as any other
+// tests that use the `custom_common` implementation since the known answers
+// depend on the exact state of `custom_common`.
```

Comment:
Oh I see this comment now, is there a reason we can't just set `COUNTER` inside the tests to some arbitrary value, then call getrandom?

I was thinking something like: https://github.com/josephlr/getrandom/commit/d8adf04b10e8b859e752d77f50d82003704d7c79

Running that seems to work as expected.

Reply:
> is there a reason we can't just set COUNTER inside the tests to some arbitrary value, then call getrandom?

That won't work. When run on a target that supports threads, the counter will be updated concurrently. WebAssembly already supports threads in some targets so I don't want to force the tests to work only in a single-threaded environment.

Reply:
Could we change the custom implementation to something that isn't order dependent then?

Maybe something which depends on the length? Like:
```rust
let mut start = buf.len() as u8;
for b in buf {
    *b = start;
    start += 1;
}
Ok(())
```

Reply:
I originally put in the order-dependent custom implementation, I now think that was probably unwise.

Reply:
The way I did this is good enough for me. I won't object to somebody rewriting this in the way they prefer.

Reply:
I can do that, just didn't want to overwrite all your work.